lib/access_pass/refresh_token_server.ex

```diff
@@ -31,13 +31,6 @@ defmodule AccessPass.RefreshTokenServer do
         {:reply, {:error, "error getting token data"}, %{}}
     end
   end
-  def return_if_exists(uniq) do
```

Comment:
Elixir wants functions with same name and arity grouped together.  In this case `return_if_exists` was separating several of the `handle_call` functions.